go.mod

```diff
@@ -2,4 +2,4 @@ module github.com/fyne-io/7guis
 
 go 1.12
 
-require fyne.io/fyne/v2 v2.0.0
+require fyne.io/fyne/v2 v2.1.2
```

Comment:
We should probably bump the minimum to Go 1.14 and update Fyne to v2.1.3 while at it.

Reply:
Sure. 2.1.3 was not out when I opened this ;).
I've not generally updated minimum go versions when opening PRs unless required, but happy to do so.

Reply:
Indeed. I took way too long to review this ;)

> I've not generally updated minimum go versions when opening PRs unless required, but happy to do so.

Isn't it required now, since Go 1.14 is the lowest supported version for 2.1.x?

Reply:
I don't think so, the definition is:

"Indicates that the module was written assuming the semantics of the Go version specified by the directive."